entity-framework/core/modeling/owned-entities.md

```diff
@@ -0,0 +1,191 @@
+---
+title: Owned Entity Types - EF Core
+author: julielerman
+ms.author: divega
+ms.date: 2/26/2018
+ms.assetid: 2B0BADCE-E23E-4B28-B8EE-537883E16DF3
+ms.technology: entity-framework-core
+uid: core/modeling/owned-entities
+---
+# Owned Entity Types
+
+>[!NOTE]
+> This feature is new in EF Core 2.0.
+
+EF Core allows you to model entity types that can only ever appear on navigation properties of other entity types. These are called _owned entity types_. The entity containing an owned entity type is its _owner_.
+
+## Explicit configuration
+
+Owned entity types are never included by EF Core in the model by convention. You can use the `OwnsOne` method in `OnModelCreating` or annotate the type with `OwnedAttrbibute` (new in EF Core 2.1) to configure the type as an owned type.
+
+In this example, StreetAddress is a type with no identity property. It is used as a property of the Order type to specify the shipping address for a particular order. In `OnModelCreating`, we use the `OwnsOne` method to specify that the ShippingAddress property is an Owned Entity of the Order type.
+
+``` csharp
+public class StreetAddress
+{
+    public string Street { get; set; }
+    public string City { get; set; }
+}
+
+public class Order
+{
+    public int Id { get; set; }
+    public StreetAddress ShippingAddress { get; set; }
+}
+
+// OnModelCreating
+modelBuilder.Entity<Order>().OwnsOne(p => p.ShippingAddress);
+```
+
+If the ShippingAddress property is private in the Order type, you can use the string version of the `OwnsOne` method:
+
+``` csharp
+modelBuilder.Entity<Order>().OwnsOne(typeof(StreetAddress), "ShippingAddress");
+```
+
+In this example, we use the `OwnedAttribute` to achieve the same goal:
+
+``` csharp
+[Owned]
+public class StreetAddress
+{
+    public string Street { get; set; }
+    public string City { get; set; }
+}
+
+public class Order
+{
+    public int Id { get; set; }
+    public StreetAddress ShippingAddress { get; set; }
+}
+```
+
+## Implicit keys
+
+In EF Core 2.0 and 2.1, only reference navigation properties can point to owned types. Collections of owned types are not supported. These reference owned types always have a one-to-one relationship with the owner, therefore they don't need their own key values. In the previous example, the StreetAddress type does need to define a key property.  
```

Comment:
Did you mean 'the StreetAddress type __doesn't__ need to define a key property.'?